src/state/CallViewModel.ts

```diff
@@ -154,11 +154,11 @@ class UserMedia {
     this.vm = new UserMediaViewModel(id, member, participant, callEncrypted);
 
     this.speaker = this.vm.speaking.pipeState(
-      // Require 1 s of continuous speaking to become a speaker, and 10 s of
+      // Require 1 s of continuous speaking to become a speaker, and 60 s of
```

Comment:
Was 10 seconds really too short?  60 feels like a lot

Reply:
It did seem to be too short in practice. In our team meetings for example, speakers often hold the floor for 30s at a time. I don't think we're at risk of the speaker section becoming overly large in large calls, because the number of people that speak during any given minute has a somewhat low upper limit - and I do think 60s has been feeling pretty good IMO!